extension/lib/shopify/product/GetProductCountAdmin.js

```diff
@@ -0,0 +1,25 @@
+class ShopifyCollectionRepositoryCommandChildrenGetCount {
```

Comment:
The name of the class doesn't fit the filename. class name should be: "ShopifyCollectionRepositoryCommandGetProductCountAdmin" I guess